src/test/java/unit/Workspace.java

```diff
@@ -124,16 +125,18 @@ void statusDescribeListReflectUpdate() throws IOException {
 
     String name = "statusDescribeListReflectUpdate";
     String description = "status list reflect update";
-    String property = "key1=value1";
+    String key = "key";
+    String value = "value";
+    String property = key + "=" + value;
 
     UFWorkspace createdWorkspace =
         WorkspaceUtils.createWorkspace(testUser, name, description, property);
 
-    // check the created workspace name, description, properties are set
+    // check the created workspace name, description, property are set
```

Comment:
should we keep the property all as `properties`?

Reply:
I changed to `property` because the test is setting one property (line 130)